tests/dummy/mirage/config.js

```diff
@@ -1,82 +1,92 @@
-import { Response } from "miragejs";
+import { Response, createServer } from "miragejs";
 
-export default function () {
-  this.namespace = "api/v1";
+export default function makeServer(config) {
+  const finalConfig = {
+    ...config,
+    routes() {
```

Comment:
This is necessary for the engine structure?

Reply:
Nope, this is the new syntax of mirage. https://github.com/miragejs/ember-cli-mirage/pull/2329